sort/tsh_sort_1_0.py

```diff
@@ -0,0 +1,64 @@
+#Some kind of a sort
+#The sort checks accounts for missing numbers
+#and last number in array being smaller
+#
+# Array to be sorted
+a = [10,2,6,7,3,5,8,1,13,15,4]
+#
+#Initalize variables
+minIndex = 0
+maxIndex = len(a)
+lowestNum = min(a)
+highestNum = max(a)
+innerIndex = 0
+outerIndex = 0
+innerBuff = 0
+buff = 0
+n = 0
+#
+#Print initial Conditions
+print "Inner Loop index is ", innerIndex

```

Comment:
Python ACT-R is using 2.7.13. It is incompatible with 3.x.x. So I am trying to learn python 2.7.x